Build failure: jit/CMakeFiles/clrjit.dir/cmake_pch.cxx.obj

I synced my local enlistment to the latest bits from today. I did a `git clean -fdx`, then when I attempted to build with `build.cmd clr+libs+libs.pretest -rc release`, it quickly fails with the following error message:

```

   Assembling: D:\runtime\src\coreclr\vm\amd64\Context.asm
  [248/1801] Linking CXX static library utilcode\utilcodestaticnohost.lib
  [249/1801] Building CXX object md\runtime\CMakeFiles\mdruntime-dbi.dir\mdfileformat.cpp.obj
  [250/1801] Building CXX object md\runtime\CMakeFiles\mdruntime-dbi.dir\liteweightstgdb.cpp.obj
  [251/1801] Generating eventpipe/eventpipehelpers.cpp, eventpipe/dotnetruntime.cpp, eventpipe/dotnetruntimerundown.cpp, eventpipe/dotnetruntimestress.cpp, eventpipe/dotnetruntimeprivate.cpp
  [252/1801] Building CXX object md\runtime\CMakeFiles\mdruntime-dbi.dir\metamodelro.cpp.obj
  [253/1801] Building CXX object md\runtime\CMakeFiles\mdruntime-dbi.dir\mdcolumndescriptors.cpp.obj
  [254/1801] Building CXX object md\runtime\CMakeFiles\mdruntime-dbi.dir\metamodel.cpp.obj
  [255/1801] Building CXX object md\runtime\CMakeFiles\mdruntime-dbi.dir\recordpool.cpp.obj
  [256/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\assemblymd.cpp.obj
  [257/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\assemblymd_emit.cpp.obj
  [258/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\classfactory.cpp.obj
  [259/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\custattr_import.cpp.obj
  [260/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\emit.cpp.obj
  [261/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\disp.cpp.obj
  [262/1801] Building CXX object jit\CMakeFiles\clrjit.dir\cmake_pch.cxx.obj
  FAILED: jit/CMakeFiles/clrjit.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_EH_FUNCLETS -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_HW_INTRINSICS -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_NO_HOST -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SIMD -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DFX_VER_INTERNALNAME_STR=clrjit.dll -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DSELF_NO_HOST -DTARGET_64BIT -DTARGET_AMD64 -DTARGET_WINDOWS -DUNICODE -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -Dclrjit_EXPORTS -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit -ID:\runtime\src\coreclr\jit -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit.dir/cmake_pch.hxx /showIncludes /Fojit\CMakeFiles\clrjit.dir\cmake_pch.cxx.obj /Fdjit\CMakeFiles\clrjit.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\CMakeFiles\clrjit.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [263/1801] Building CXX object jit\CMakeFiles\clrjit_universal_arm64_x64.dir\cmake_pch.cxx.obj
  FAILED: jit/CMakeFiles/clrjit_universal_arm64_x64.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_EH_FUNCLETS -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_HW_INTRINSICS -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_MULTIREG_RETURN -DFEATURE_NO_HOST -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SIMD -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DFX_VER_INTERNALNAME_STR=clrjit_universal_arm64_x64.dll -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DSELF_NO_HOST -DTARGET_64BIT -DTARGET_ARM64 -DUNICODE -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -Dclrjit_universal_arm64_x64_EXPORTS -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit -ID:\runtime\src\coreclr\jit -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_universal_arm64_x64.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_universal_arm64_x64.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_universal_arm64_x64.dir/cmake_pch.hxx /showIncludes /Fojit\CMakeFiles\clrjit_universal_arm64_x64.dir\cmake_pch.cxx.obj /Fdjit\CMakeFiles\clrjit_universal_arm64_x64.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\CMakeFiles\clrjit_universal_arm64_x64.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [264/1801] Building CXX object md\compiler\CMakeFiles\mdcompiler_dac.dir\custattr_emit.cpp.obj
  [265/1801] Building CXX object jit\CMakeFiles\clrjit_win_x64_x64.dir\cmake_pch.cxx.obj
  FAILED: jit/CMakeFiles/clrjit_win_x64_x64.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_EH_FUNCLETS -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_HW_INTRINSICS -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_NO_HOST -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SIMD -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DFX_VER_INTERNALNAME_STR=clrjit_win_x64_x64.dll -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DSELF_NO_HOST -DTARGET_64BIT -DTARGET_AMD64 -DTARGET_WINDOWS -DUNICODE -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -Dclrjit_win_x64_x64_EXPORTS -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit -ID:\runtime\src\coreclr\jit -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_win_x64_x64.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_win_x64_x64.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_win_x64_x64.dir/cmake_pch.hxx /showIncludes /Fojit\CMakeFiles\clrjit_win_x64_x64.dir\cmake_pch.cxx.obj /Fdjit\CMakeFiles\clrjit_win_x64_x64.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\CMakeFiles\clrjit_win_x64_x64.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [266/1801] Building CXX object jit\CMakeFiles\clrjit_universal_arm_x64.dir\cmake_pch.cxx.obj
  FAILED: jit/CMakeFiles/clrjit_universal_arm_x64.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DARM_SOFTFP -DCONFIGURABLE_ARM_ABI -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_EH_FUNCLETS -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_NO_HOST -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DFX_VER_INTERNALNAME_STR=clrjit_universal_arm_x64.dll -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DSELF_NO_HOST -DTARGET_ARM -DUNICODE -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -Dclrjit_universal_arm_x64_EXPORTS -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit -ID:\runtime\src\coreclr\jit -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_universal_arm_x64.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_universal_arm_x64.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_universal_arm_x64.dir/cmake_pch.hxx /showIncludes /Fojit\CMakeFiles\clrjit_universal_arm_x64.dir\cmake_pch.cxx.obj /Fdjit\CMakeFiles\clrjit_universal_arm_x64.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\CMakeFiles\clrjit_universal_arm_x64.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [267/1801] Building CXX object jit\CMakeFiles\clrjit_win_x86_x64.dir\cmake_pch.cxx.obj
  FAILED: jit/CMakeFiles/clrjit_win_x86_x64.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_HW_INTRINSICS -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_NO_HOST -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SIMD -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DFX_VER_INTERNALNAME_STR=clrjit_win_x86_x64.dll -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DSELF_NO_HOST -DTARGET_WINDOWS -DTARGET_X86 -DUNICODE -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -Dclrjit_win_x86_x64_EXPORTS -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit -ID:\runtime\src\coreclr\jit -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_win_x86_x64.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_win_x86_x64.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_win_x86_x64.dir/cmake_pch.hxx /showIncludes /Fojit\CMakeFiles\clrjit_win_x86_x64.dir\cmake_pch.cxx.obj /Fdjit\CMakeFiles\clrjit_win_x86_x64.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\CMakeFiles\clrjit_win_x86_x64.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [268/1801] Building CXX object jit\static\CMakeFiles\clrjit_obj.dir\cmake_pch.cxx.obj
  FAILED: jit/static/CMakeFiles/clrjit_obj.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_EH_FUNCLETS -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_HW_INTRINSICS -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SIMD -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DTARGET_64BIT -DTARGET_AMD64 -DTARGET_WINDOWS -DUNICODE -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\static -ID:\runtime\src\coreclr\jit\static -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc -ID:\runtime\src\coreclr\jit /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/static/CMakeFiles/clrjit_obj.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/static/CMakeFiles/clrjit_obj.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/static/CMakeFiles/clrjit_obj.dir/cmake_pch.hxx /showIncludes /Fojit\static\CMakeFiles\clrjit_obj.dir\cmake_pch.cxx.obj /Fdjit\static\CMakeFiles\clrjit_obj.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\static\CMakeFiles\clrjit_obj.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [269/1801] Building CXX object jit\CMakeFiles\clrjit_unix_x64_x64.dir\cmake_pch.cxx.obj
  FAILED: jit/CMakeFiles/clrjit_unix_x64_x64.dir/cmake_pch.cxx.obj
  C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1431~1.311\bin\Hostx64\x64\cl.exe  /nologo /TP -DDEBUGGING_SUPPORTED -DEnC_SUPPORTED -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_BASICFREEZE -DFEATURE_CODE_VERSIONING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_COMINTEROP -DFEATURE_COMINTEROP_APARTMENT_SUPPORT -DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION -DFEATURE_COMWRAPPERS -DFEATURE_CORECLR -DFEATURE_DATABREAKPOINT -DFEATURE_DEFAULT_INTERFACES -DFEATURE_EH_FUNCLETS -DFEATURE_ENC_SUPPORTED -DFEATURE_EVENT_TRACE -DFEATURE_HIJACK -DFEATURE_HW_INTRINSICS -DFEATURE_ICASTABLE -DFEATURE_INSTANTIATINGSTUB_AS_IL -DFEATURE_INTEROP_DEBUGGING -DFEATURE_ISYM_READER -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES -DFEATURE_MULTICASTSTUB_AS_IL -DFEATURE_MULTICOREJIT -DFEATURE_MULTIREG_RETURN -DFEATURE_NO_HOST -DFEATURE_ON_STACK_REPLACEMENT -DFEATURE_PERFTRACING -DFEATURE_PGO -DFEATURE_PORTABLE_SHUFFLE_THUNKS -DFEATURE_PROFAPI_ATTACH_DETACH -DFEATURE_READYTORUN -DFEATURE_REJIT -DFEATURE_SIMD -DFEATURE_SPECIAL_USER_MODE_APC -DFEATURE_STANDALONE_GC -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_TIERED_COMPILATION -DFEATURE_TYPEEQUIVALENCE -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP -DFEATURE_WIN32_REGISTRY -DFX_VER_INTERNALNAME_STR=clrjit_unix_x64_x64.dll -DHOST_64BIT -DHOST_AMD64 -DHOST_WINDOWS -DJIT_BUILD -DNDEBUG -DPROFILING_SUPPORTED -DSELF_NO_HOST -DTARGET_64BIT -DTARGET_AMD64 -DTARGET_UNIX -DUNICODE -DUNIX_AMD64_ABI -DUNIX_AMD64_ABI_ITF -DURTBLDENV_FRIENDLY=Retail -DWIN32 -DWIN32_LEAN_AND_MEAN -DWINVER=0x0602 -D_BLD_CLR -D_CRT_SECURE_NO_WARNINGS -D_FILE_OFFSET_BITS=64 -D_SECURE_SCL=0 -D_UNICODE -D_WIN32 -D_WIN32_WINNT=0x0602 -Dclrjit_unix_x64_x64_EXPORTS -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit -ID:\runtime\src\coreclr\jit -ID:\runtime\src\native -ID:\runtime\src\coreclr\pal\prebuilt\inc -ID:\runtime\artifacts\obj -ID:\runtime\src\coreclr\inc -ID:\runtime\src\coreclr\debug\inc -ID:\runtime\src\coreclr\debug\inc\amd64 -ID:\runtime\src\coreclr\debug\inc\dump -ID:\runtime\src\coreclr\md\inc -ID:\runtime\src\coreclr\classlibnative\bcltype -ID:\runtime\src\coreclr\classlibnative\cryptography -ID:\runtime\src\coreclr\classlibnative\inc -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc -ID:\runtime\src\coreclr\hosts\inc -ID:\runtime\src\coreclr\minipal -ID:\runtime\artifacts\obj\coreclr\windows.x64.Release\inc\etw -ID:\runtime\src\coreclr\jit\.\jitstd -ID:\runtime\src\coreclr\jit\..\inc /DWIN32 /D_WINDOWS    /GR- /guard:cf /guard:ehcont /O2 /Ob2 /DNDEBUG -MT /Ox /nologo /W4 /WX /Oi /Oy- /Gm- /Zp8 /Gy /GS /fp:precise /FC /MP /Zm200 /Zc:strictStrings /Zc:wchar_t /Zc:inline /Zc:forScope /wd4065 /wd4100 /wd4127 /wd4131 /wd4189 /wd4200 /wd4201 /wd4206 /wd4239 /wd4244 /wd4245 /wd4291 /wd4310 /wd4324 /wd4366 /wd4456 /wd4457 /wd4458 /wd4459 /wd4463 /wd4505 /wd4702 /wd4706 /wd4733 /wd4815 /wd4838 /wd4918 /wd4960 /wd4961 /wd5105 /wd5205 /we4007 /we4013 /we4102 /we4551 /we4700 /we4640 /we4806 /w34092 /w34121 /w34125 /w34130 /w34132 /w34212 /w34530 /w35038 /w44177 /Zi /ZH:SHA_256 /source-charset:utf-8 /GL /EHa /Zl /permissive- /YcD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_unix_x64_x64.dir/cmake_pch.hxx /FpD:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_unix_x64_x64.dir/./cmake_pch.cxx.pch /FID:/runtime/artifacts/obj/coreclr/windows.x64.Release/jit/CMakeFiles/clrjit_unix_x64_x64.dir/cmake_pch.hxx /showIncludes /Fojit\CMakeFiles\clrjit_unix_x64_x64.dir\cmake_pch.cxx.obj /Fdjit\CMakeFiles\clrjit_unix_x64_x64.dir\ /FS -c D:\runtime\artifacts\obj\coreclr\windows.x64.Release\jit\CMakeFiles\clrjit_unix_x64_x64.dir\cmake_pch.cxx
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): error C2665: 'operator new': none of the 6 overloads could convert all the argument types
  D:\runtime\src\coreclr\inc\utilcode.h(4447): note: could be 'void *operator new(size_t,const CExecutable &,const NoThrow &)'
  D:\runtime\src\coreclr\jit\jitstd\vector.h(670): note: while trying to match the argument list '(unsigned __int64, T *, jitstd::placement_t)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\jitstd\vector.h(662): note: while compiling class template member function 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)'
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(681): note: see reference to function template instantiation 'void jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>::resize(jitstd::vector<T,jitstd::allocator<T>>::size_type,const T &)' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  D:\runtime\src\coreclr\jit\loopcloning.h(667): note: see reference to class template instantiation 'jitstd::vector<JitExpandArrayStack<LcOptInfo *> *,jitstd::allocator<T>>' being compiled
          with
          [
              T=JitExpandArrayStack<LcOptInfo *> *
          ]
  [270/1801] Building CXX object vm\wks\CMakeFiles\cee_wks_core.dir\cmake_pch.cxx.obj
  [271/1801] Building CXX object vm\eventing\eventpipe\CMakeFiles\eventpipe_objs.dir\cmake_pch.cxx.obj
  [272/1801] Building CXX object vm\wks\CMakeFiles\cee_wks_obj.dir\cmake_pch.cxx.obj
  [273/1801] Building CXX object vm\wks\CMakeFiles\cee_wks_mergeable_obj.dir\cmake_pch.cxx.obj
  [274/1801] Building CXX object vm\eventing\eventpipe\CMakeFiles\eventpipe_gen_objs.dir\cmake_pch.cxx.obj
  [275/1801] Building CXX object vm\CMakeFiles\cee_dac.dir\cmake_pch.cxx.obj
  ninja: build stopped: subcommand failed.
```

Let me know if I can provide more info/details from my build configuration.

--- a/src/coreclr/jit/CMakeLists.txt
+++ b/src/coreclr/jit/CMakeLists.txt
@@ -11,10 +11,6 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_
   add_compile_options(-Wno-error)
 endif()
 
-if (MSVC)
-  add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/permissive->)
-endif()
-
 function(create_standalone_jit)
 
   set(oneValueArgs TARGET OS ARCH)

--- a/src/coreclr/jit/unwind.h
+++ b/src/coreclr/jit/unwind.h
@@ -79,6 +79,32 @@ protected:
     {
     }
 
+// TODO: How do we get the ability to access uwiComp without error on Clang?
+#if defined(DEBUG) && !defined(__GNUC__)
+
+    template <typename T>
+    T dspPtr(T p)
+    {
+        return uwiComp->dspPtr(p);
+    }
+
+    template <typename T>
+    T dspOffset(T o)
+    {
+        return uwiComp->dspOffset(o);
+    }
+
+    static const char* dspBool(bool b)
+    {
+        return (b) ? "true" : "false";
+    }
+
+#endif // DEBUG
+
+    //
+    // Data
+    //
+
     Compiler* uwiComp;
 };
 

